docs/tutorials/multimodal/multilingual_distillation.md

```diff
@@ -0,0 +1,185 @@
+
+During the deployment of neural networks, limited resources always
+prevent us from using a larger models for better performance. 
+To benefit from large models under this constraint, 
+knowledge distillation asks a small student model to learn from a large teacher model.
+In this way, the small student model can be practically deployed under real-world constraints,
+while in the mean time more knowledge is captured by the large teacher model.
+
+In this tutorial, we introduce how `MultiModalPredictor` can help you complete a distillation task. For the purpose of demonstration, 
+we use the [Cross-Lingual Paraphrase Adversaries from Word Scrambling (PAWS-X)](https://arxiv.org/pdf/1908.11828.pdf) dataset, 
+which conprises 23,659 human translated PAWS evaluation pairs in six typologically distinct languages:
+French, Spanish, German, Chinese, Japanese, and Korean. 
+We will demonstrate how to use a large model to guide the learning 
+and improve the performance of a small model in Autogluon.
+
+## Load Dataset
+
+The [Cross-Lingual Paraphrase Adversaries from Word Scrambling (PAWS-X)](https://arxiv.org/pdf/1908.11828.pdf) dataset contains 
+sentence pairs in six languages. 
+Here, we use the English and German to demonstrate.
+In the label column, `0` means different meaning for the pair of sentences and `1` means same meaning.
+
+```{.python .input}
+def getDatasetSplits(pawsx_tasks = ["en", "de"]):
+    train_dfs = {}
+    val_dfs = {}
+    test_dfs = {}
+    for task in pawsx_tasks:
+        dataset = load_dataset("paws-x", task)
+        train_dfs[task] = dataset["train"].to_pandas()
+        val_dfs[task] = dataset["validation"].to_pandas()
+        test_dfs[task] = dataset["test"].to_pandas()
+        print(
+            "task %s: train %d, val %d, test %d"
+            % (task, len(train_dfs[task]), len(val_dfs[task]), len(test_dfs[task]))
+        )
+    train_df = pd.concat(train_dfs)
+    val_df = pd.concat(val_dfs)
+    test_dfs["all"] = pd.concat(test_dfs)
+
+    return train_df, val_df, test_dfs
+    
+train_df, val_df, test_dfs = getDatasetSplits()
+```
+
+## Finetune the large teacher model
+
+We first finetune a large teacher model [DeBERTaV3](https://arxiv.org/abs/2111.09543). 
+Since `MultiModalPredictor` integrates with the [Huggingface/Transformers](https://huggingface.co/docs/transformers/index) 
+(as explained in :ref:`sec_textprediction_customization`), 
+we directly load the multilingual DeBERTaV3 model pretrained by Microsoft from Huggingface/Transformers, 
+with the key as [microsoft/mdeberta-v3-base](https://huggingface.co/microsoft/mdeberta-v3-base).
+
+```{.python .input}
+teacher_predictor = MultiModalPredictor(label="label", eval_metric="accuracy")
+teacher_predictor.fit(
+    train_df,
+    tuning_data=val_df,
+    hyperparameters={
+        "model.hf_text.checkpoint_name": "microsoft/mdeberta-v3-base",
+        "env.precision": "32",
+        "optimization.max_epochs": 8,
```

Comment:
How long does the training take? Does it have an impact on the doc building time?